docker/Dockerfile-ubuntu-22.04

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04 as base
+
+RUN DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" \
```

Comment:
Do we not need to build cMake and grpc like how we are doing in ubuntu-20.04 docker file?

Reply:
Ubuntu-22.04 needs to be updated

Reply:
I can work on the ubuntu 22.04 Docker file but I may need a day or two to complete it.